tests/testapp/test_aggregates.py

```diff
@@ -104,14 +103,14 @@ def test_separator_big(self):
     def test_expression(self):
         concat = GroupConcat(F("id") + 1)
         out = self.shakes.tutees.aggregate(tids=concat)
-        concatted_ids = ",".join([str(self.jk.id + 1), str(self.grisham.id + 1)])
+        concatted_ids = [str(self.jk.id + 1), str(self.grisham.id + 1)]
         assert out == {"tids": concatted_ids}
 
     def test_application_order(self):
         out = Author.objects.exclude(id=self.shakes.id).aggregate(
             tids=GroupConcat("tutor_id", distinct=True)
         )
-        assert out == {"tids": str(self.shakes.id)}
+        assert out == {"tids": set(map(str,[self.shakes.id]))}
```

Comment:
```suggestion
        assert out == {"tids": {str(self.shakes.id)}}
```

Reply:
Thanks. I'm quite old school and sets are somewhat new to me ...